### ISSUE TITLE
Courses Drop-Down Requires Major Refactor

### ISSUE THREAD

**Issue by aopell** [aopell/SchoologyPlus#107]
The courses drop-down has been entirely redesigned, and we need to make quite a few changes to get it working as well as it did before.

- [x] Course Icons (potentially resize the tiles?)
- [x] Archived Courses button (requires a complete rewrite using a MutationObserver since course drop-down content is created when the button is clicked)
- [ ] Sorting (not sure how this is handled since I don't have classes, though I might steal my sister's account for debugging stuff)

**Comment by aopell** [comment]
@glen3b New issue: Archived courses link does not show up when window width is less than 1460px

**Comment by glen3b** [comment]
@aopell I changed the selector, it seems to work now, although I can't say I fully understand my change.

Regarding the other two points on this ticket:
- I've made progress toward addressing icons and overall restyling the tiles in this dropdown, but I want @aopell to take a look at it.
- Sorting is doable within Schoology: on the "My Courses" page linked from the dropdown (`/courses`), there's a "Reorder Courses" button, and that order is what's used in the dropdown. I believe this order is also used by the course dashboard accessible via the homepage (which provides a drag-and-drop interface to change it). I don't think we want to override this order in the dropdown, but it's certainly less readily accessible than the little arrow button in the old courses dropdown. If we want to readd that button and open the new reorder courses dialog, if we can find a good place for it, I think that would be cool but difficult. Besides that, I don't know if we want to touch the UI at all. I do think making that reorderability more discoverable would be good, but I don't know how to do that well in a practical manner.
